fetools-app/src/pages/CharacterFinder.jsx

```diff
@@ -105,7 +120,10 @@ function CharacterFinder() {
           (character.hex &&
             character.hex.toLowerCase().includes(query.toLowerCase())) ||
           (character.unicode &&
-            character.unicode.toLowerCase().includes(query.toLowerCase()))
+            character.unicode.toLowerCase().includes(query.toLowerCase())) ||
+          (character.css &&
+            (character.css.toLowerCase().includes(query.toLowerCase()) ||
+              character.css.toLowerCase().includes('\\' + query.toLowerCase())))
```

Comment:
The second condition is never true unless the first is also true.

```suggestion
            (character.css.toLowerCase().includes(query.toLowerCase()))
          )
```